jobs/python.setup.py.yml

```diff
@@ -61,13 +61,26 @@ jobs:
   steps:
   - template: ../steps/python/python.build.yml
 
+jobs:
+- job: 'ReleaseChecks'
+  dependsOn:
+    - macOS
+    - Linux
+    - Windows
+  pool:
+    vmImage: 'ubuntu-16.04'
+  condition: and(succeeded('macOS'), succeeded('Windows'), succeeded('Linux'), succeeded('ReleaseChecks'), 'PullRequest')
```

Comment:
The check for a PR should be like this `eq(variables['build.reason'], 'PullRequest')`